### PR TITLE
feat: add --quite mode for nuxt-build

### LINKED ISSUE #3699: nuxt build --quiet
### What problem does this feature solve?

Silence the output of nuxt so when run in ci or invoked by other processes the default output can be kept clean and unpolluted.

### What does the proposed changes look like?

Add a --quiet flag which silences all default output to stdout.

<div align="right"><sub><em>This feature request is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/nuxt.js/issues/c7542">#c7542</a>)</em></sub></div>

### PR DESCRIPTION
**Pull request by TheAlexLichter**
resolves #3699

### CODE REVIEW

**Review by clarkdo — COMMENTED** [review]
https://github.com/nuxt/nuxt.js/commit/c4792d247b891b005282006a916a9f26946eeb46

**Comment by manniL** [comment]
@clarkdo Oh no :see_no_evil:  Why I didn't notice that commit while writing the PR :thinking: 

 Shouldn't `options.test` exclude webpackbar then as well?

**Comment by clarkdo** [comment]
I think no need because webpackbar will clear console, you can have a look at appveyor , there is no any building info log

**Comment by manniL** [comment]
Agreed. Have noticed this locally but wasn't unsure if CI services handle it correctly as well.

Then at least the command option?

**Comment by clarkdo** [comment]
Sure, is slient a legacy option?

**Comment by manniL** [comment]
What do you mean with `legacy option` :thinking:

**Comment by clarkdo** [comment]
Existed option?

**Comment by manniL** [comment]
Don't think so, but the `s` shorthand is already blocked by `--spa`

**Comment by clarkdo** [comment]
It slient is not existed option, I think build.client is more proper and default value is false.

**Comment by manniL** [comment]
So you mean `build.client` is false by default and will be true if either `test` is set (f.ex. because of env) or `nuxt-build` has been executed with `--quiet`?

**Comment by clarkdo** [comment]
If it’s betters to move level into options.js, in bin only set test if quiet

**Comment by manniL** [comment]
Right, better separation. I've included your change suggestions :relaxed:

**Comment by manniL** [comment]
@clarkdo Thanks for your helpful guidance :pray:

**Comment by clarkdo** [comment]
Can you reset commit to let ci rerun?

**Review by clarkdo — APPROVED** [review]

**Comment by manniL** [comment]
Did it once, but it's still failing :thinking: 
One more time.

File: bin/nuxt-build [hunk 1]
```diff
@@ -47,6 +49,11 @@ if (argv.analyze && typeof options.build.analyze !== 'object') {
   options.build.analyze = true
 }
```
Comment (clarkdo) [inline bin/nuxt-build]: quiet

**Comment by manniL** [comment]
@clarkdo Have to adapt tests as consola won't print anything now because we changed the log level :see_no_evil:

**Comment by clarkdo** [comment]
actually consola has checked if ci or test， then use minimum way for logging

**Comment by manniL** [comment]
@clarkdo But it won't recognize it if we use `--quite` I guess.

**Comment by clarkdo** [comment]
Just revert the version set level in bin，I’ll consider future refactor

**Pull request merged**